systemd-id128: add page (#11514)
tldr-pages/tldr: pages/linux/systemd-id128.md

# systemd-id128

> Generate and print sd-128 identifiers.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-id128.html>.

- Generate a new random identifier:

> `$ systemd-id128 new`

Author: fixed

> - Print the identifier of the current machine:
> 
> `systemd-id128 {{machine-id}}`
> 
> - Print the identifier of the current boot:
> 
> `systemd-id128 {{boot-id}}`

Reviewer: Subcommands are not placeholders.
```suggestion
- Print the identifier of the current machine:

`systemd-id128 machine-id`

- Print the identifier of the current boot:

`systemd-id128 boot-id`
```

> - Print the identifier of the current service invocation:
> 
> `systemd-id128 {{invocation-id}}`

Reviewer: ```suggestion
- Print the identifier of the current service invocation (this is available in systemd services):

`systemd-id128 invocation-id`
```

> - Generate a new random identifier and print it in UUID format:

Reviewer: ```suggestion
- Generate a new random identifier and print it as an UUID (five groups of digits separated by hyphens):
```

> `systemd-id128 new -u`

Reviewer: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax
```suggestion
`systemd-id128 new --uuid`
```
